frontend/components/TableContainer/DataTable/HostMdmStatusCell/HostMdmStatusCell.tsx

```
@@ -1,11 +1,12 @@
 import React from "react";
-import { DEFAULT_EMPTY_CELL_VALUE } from "utilities/constants";
+import { DEFAULT_EMPTY_CELL_VALUE, MDM_STATUS_TOOLTIP } from "utilities/constants";
```

ESLint

[error] Replace `·DEFAULT_EMPTY_CELL_VALUE,·MDM_STATUS_TOOLTIP·` with `⏎··DEFAULT_EMPTY_CELL_VALUE,⏎··MDM_STATUS_TOOLTIP,⏎`

(prettier/prettier)